src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -426,6 +436,20 @@ namespace ts.server.typingsInstaller {
             }
         }
 
+        private writePackageVersion(cachePath: string) {
+            const packageJson = combinePaths(cachePath, "package.json");
+            const npmConfig = <NpmConfig>JSON.parse(this.installTypingHost.readFile(packageJson));
+            npmConfig.typeScriptVersion = ts.version;
+            this.installTypingHost.writeFile(packageJson, JSON.stringify(npmConfig));
+            if (this.log.isEnabled()) {
+                this.log.writeLine(`Wrote version ${ts.version} to package.json`);
+            }
+        }
+
+        private areVersionsEqualMajorMinor(version1: string, version2: string) {

```

Comment:
we do something similar for @ts tags, see `versionMajorMinor` can we unify the version manipulation in one place.